rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -19,7 +19,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}rubygems
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
 Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.1
```

Comment:
Downgraded to `0.8` (https://github.com/theforeman/foreman_ansible/blob/1.4.4/foreman_ansible.gemspec, https://github.com/theforeman/foreman_ansible/blob/1.4.5/foreman_ansible.gemspec)